lib/linter-eslint.coffee

```diff
@@ -44,7 +53,7 @@ module.exports =
 
   provideLinter: ->
     provider =
-      grammarScopes: ['source.js', 'source.js.jsx', 'source.babel', 'source.js-semantic']
+      grammarScopes: atom.config.get('linter-eslint.fileScopes').split(',')
```

Comment:
You should be observing the setting in activate() instead of get()ing it every time.